### PR TITLE
Fix the broken link to the contribution graph

### LINKED ISSUE #13: Contribution graph is not show anymore
It seems that the link to the contribution graph on the org page is broken.

### PR DESCRIPTION
**Pull request by choeppler**

### CODE REVIEW

**Comment by strieflin** [comment]
Hi @choeppler ,

Welcome to the Carbyne Stack community and thank you for submitting your first pull request! 🎉 We really appreciate you taking the time to contribute.

The fact that the contribution graph doesn't show up anymore is somewhat mysterious. The `.svg` file is and has always been created in the root folder of this repository by the `herdstat` tool. It seems that the link embedded in the `README.md` within the `profile` folder is pointing and always has been pointing to the (non-existing) file in the `profile` folder. It is a mystery to me how this has ever worked. But it has.

The fix you have proposed will unfortunately not work as the contribution graph is generated by a nightly workflow in the root folder. I think the better approach is to update the link in the `README.md` file to point to the file in the root folder, i.e., from `https://github.com/carbynestack/.github/blob/master/profile/contribution-graph.svg` to `https://github.com/carbynestack/.github/blob/master/contribution-graph.svg`.

Would you like to update your PR accordingly?

**Comment by strieflin** [comment]
You should also add yourself as a contributor to the `NOTICE.md` file.

**Comment by strieflin** [comment]
Created #13 to track this issue.

**Comment by choeppler** [comment]
> The fix you have proposed will unfortunately not work as the contribution graph is generated by a nightly workflow in the root folder.

That's why I propose to patch [.github/workflows/herdstat.yaml](https://github.com/carbynestack/.github/pull/12/files#diff-7044b0fd33aaec62ea6b077a40afca9b29e80dc09402b4425bbf1a66425fc420). Or is there another workflow you are referring to?

File: .github/workflows/herdstat.yaml [hunk 1]
```diff
@@ -32,5 +32,5 @@ jobs:
         with:
           author_name: cs-minion
           author_email: rng_cr_carbynestack@bosch.com
-          add: 'contribution-graph.svg'
+          add: 'profile/contribution-graph.svg'
```
Comment (strieflin) [inline .github/workflows/herdstat.yaml:35]: This not the new file created by the nightly `herdstat` run but the one that will exist in the `profile` folder as a result of merging your PR.

**Comment by strieflin** [comment]
> Or is there another workflow you are referring to?

No, that is the workflow I am talking about. My point is that your PR moves the _existing_ file `contribution-graph.svg` from the `root` folder into the `profile` folder. This means that this file will never be updated anymore. Instead a new version of `contribution-graph.svg` will be created by the workflow in the root folder again. The Github org page, however, will forever show the file moved by the PR and not the new one. See also https://github.com/carbynestack/.github/pull/12/files#r1815059670.

File: .github/workflows/herdstat.yaml (hunk 1 above)
Comment (choeppler) [reply .github/workflows/herdstat.yaml:35]: Right. I also updated [.herdstat.yaml](https://github.com/carbynestack/.github/pull/12/files#diff-50eab34b05ede56ab86c6ec547baf3dcb7f22601e39b04122a8617dc7a060e09) so the file will (hopefully) be created in the profile folder.

**Comment by strieflin** [comment]
I manually verified that the approach works. Luckily, the author of herdstat made the tool flexible enough :wink:. Will merge this after you have added yourself to the `NOTICE.md` file.

**Review by strieflin — CHANGES_REQUESTED** [review]
LGTM. However, there is a conflict again from the newly generated `contribution-graph.svg` file again. We have to resolve the conflict and merge the PR the same day 😁.

**Comment by choeppler** [comment]
Can't you do a rebase and merge? Well, I can do so too, of course, and force update the branch...

**Comment by strieflin** [comment]
> Can't you do a rebase and merge? Well, I can do so too, of course, and force update the branch...

I don't think so. `choeppler:master` is your repo/branch to which I don't have access to. Although, I don't really know. Maybe it is possible somehow.

**Comment by strieflin** [comment]
Works again. Thank you @choeppler.

**Pull request merged**